Source/WebKit/UIProcess/API/Cocoa/NSAttributedString.mm

```diff
@@ -133,12 +136,27 @@ @implementation _WKAttributedStringWebViewCache
     return configuration;
 }
 
++ (NSMutableArray<NSURL *> *)readOnlyAccessPaths
+{
+    static auto* readOnlyAccessPaths = [[NSMutableArray alloc] initWithCapacity:maximumReadOnlyAccessPaths];
```

Comment:
This should at least use a `RetainPtr<NSMutableArray>`, and maybe a `NeverDestroyed<RetainPtr<NSMutableArray>>`.

It should not be a raw pointer.

Reply:
Will fix.

Reply:
Done!